notebooks/example_notebook_error_labeling.ipynb

```diff
@@ -0,0 +1,240 @@
+    "            \"unknown\", \"stop sign\", \"parking meter\", \"bench\", \"bird\", \"cat\", \"dog\", \"horse\",\n",
+    "            \"sheep\", \"cow\", \"elephant\", \"bear\", \"zebra\", \"giraffe\", \"unknown\", \"backpack\",\n",
+    "            \"umbrella\", \"unknown\", \"unknown\", \"handbag\", \"tie\", \"suitcase\", \"frisbee\", \"skis\",\n",
+    "            \"snowboard\", \"sports ball\", \"kite\", \"baseball bat\", \"baseball glove\", \"skateboard\",\n",
+    "            \"surfboard\", \"tennis racket\", \"bottle\", \"unknown\", \"wine glass\", \"cup\", \"fork\", \"knife\",\n",
+    "            \"spoon\", \"bowl\", \"banana\", \"apple\", \"sandwich\", \"orange\", \"broccoli\", \"carrot\", \"hot dog\",\n",
+    "            \"pizza\", \"donut\", \"cake\", \"chair\", \"couch\", \"potted plant\", \"bed\", \"unknown\", \"dining table\",\n",
+    "            \"unknown\", \"unknown\", \"toilet\", \"unknown\", \"tv\", \"laptop\", \"mouse\", \"remote\", \"keyboard\",\n",
+    "            \"cell phone\", \"microwave\", \"oven\", \"toaster\", \"sink\", \"refrigerator\", \"unknown\",\n",
+    "            \"book\", \"clock\", \"vase\", \"scissors\", \"teddy bear\", \"hair drier\", \"toothbrush\" ])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def highlight_cells(value):\n",
+    "    color = 'background-color: yellow'  # Set the desired highlight color\n",
+    "\n",
+    "    if value == ErrorLabelType.MATCH:\n",
+    "        return color\n",
+    "    else:\n",
+    "        return ''"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def remove_first_last(array):\n",
+    "    result = []\n",
+    "    # for sublist in array:\n",
+    "    if len(array) > 2:\n",
+    "        result.append(array[1:-1])\n",
+    "    return result\n",
+    "\n",
+    "def draw_detections(base_img, detections, gt):\n",
+    "    img_copy = copy.deepcopy(base_img)\n",
+    "    im = ImageDraw.Draw(img_copy)\n",
+    "    for idx, detection in enumerate(detections):\n",
+    "        d = detection[1:-1]\n",
+    "        im.rectangle(((d[0], d[1]), (d[2], d[3])), outline=\"green\", width=2)\n",
+    "        im.text((d[0], d[1]), text=\"p\"+str(idx), fill=\"green\", stroke_width=4, stroke_fill=\"white\", font=ImageFont.truetype(\"arial.ttf\", 12))\n",
+    "    for idx, detection in enumerate(gt):\n",
+    "        d = detection[1:-1]\n",
+    "        im.rectangle(((d[0], d[1]), (d[2], d[3])), outline=(255, 0, 255), width=2)\n",
+    "        im.text((d[2]-20, d[1]), text=\"gt\"+str(idx), fill=(255, 0, 255), stroke_width=4, stroke_fill=\"white\", font=ImageFont.truetype(\"arial.ttf\", 12))\n",
+    "    return img_copy\n",
+    "\n",
+    "detection_model = WrappedObjectDetectionModel(model=model, number_of_classes = 90)\n",
```

Comment:
Shouldn't this be done via wrap_model call instead?  That's the more user-facing API?

Reply:
Fixed!